keycloak/lib/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#
+#RUN /opt/keycloak/bin/kc.sh build
+#
+#FROM quay.io/keycloak/keycloak:22.0.4
+#ENTRYPOINT ["/opt/keycloak/bin/kc.sh"]
+
+FROM quay.io/keycloak/keycloak:latest as builder
```

Comment:
Should we pin this to a version?

Reply:
This is definitely correct, thanks for the catching